appveyor.yml

```diff
@@ -6,6 +6,24 @@ stack: go 1.14
 before_test:
   - choco install mingw
   - choco install make
+  - choco install wixtoolset
+  - set PATH=%PATH%;"C:\Program Files (x86)\WiX Toolset v3.11\bin"
+  - make MOCK_BUNDLE=true BUNDLE_DIR=./ out/windows-amd64/crc-installer.zip
   - make cross
+  
 test_script:
   - make test
+
+artifacts:
+  - path: out\linux-amd64\crc
+    name: crc linux
+    type: File
+  - path: out\darwin-amd64\crc
+    name: crc darwin
+    type: File
+  - path: out\windows-amd64\crc.exe
+    name: crc windows
+    type: File
+  - path: out\windows-amd64\crc-installer.zip
+    name: crc msi installer
```

Comment:
it would still be windows

`crc windows installer`

and call the regular release perhaps as

`crc windows portable`

Reply:
Changed, but these are only for PR builds.